### PR TITLE
fix(@desktop/wallet): Fixing bugs found during demo

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
### What does the PR do

Fixing bugs found during demo-

1. After sending a collectible a normal tx doesnt work because the chains are locked 
2. Locked in amounts are not getting reset on closing on menu
3. Hide collectibles when on bridge 
4. Resizing issue on send modal
5. Bridge routes not visible everytime

### Affected areas

SendModal

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

### CODE REVIEW

File: src/app/modules/main/wallet_section/send/network_model.nim [hunk 1]
```diff
@@ -148,6 +148,10 @@ QtObject:
       self.items[i].amountOut = ""
       self.items[i].resetToNetworks()
       self.items[i].hasGas = true
+      self.items[i].isEnabled = true
+      self.items[i].isPreferred = true
+      self.items[i].locked =  false
+      self.items[i].lockedAmount = ""
       self.dataChanged(index, index, @[ModelRole.AmountIn.int, ModelRole.ToNetworks.int, ModelRole.HasGas.int, ModelRole.AmountOut.int])
```
Comment (caybro) [inline src/app/modules/main/wallet_section/send/network_model.nim:155]: You also have to signal the changed properties here

File: ui/imports/shared/popups/send/views/TokenListView.qml [hunk 2]
```diff
@@ -162,7 +162,7 @@ Item {
     Component {
         id: tokenHeader
         SearchBoxWithRightIcon {
-            showTopBorder: true
+            showTopBorder: root.onlyAssets ? false : true
```
Comment (caybro) [inline ui/imports/shared/popups/send/views/TokenListView.qml:165]:
```suggestion
            showTopBorder: !root.onlyAssets
```

**Review by dlipicar — APPROVED** [review]

File: src/app/modules/main/wallet_section/send/network_model.nim [hunk 3]
```diff
@@ -148,7 +148,13 @@ QtObject:
...
       self.items[i].resetToNetworks()
       self.items[i].hasGas = true
-      self.dataChanged(index, index, @[ModelRole.AmountIn.int, ModelRole.ToNetworks.int, ModelRole.HasGas.int, ModelRole.AmountOut.int])
+      self.items[i].isEnabled = true
+      self.items[i].isPreferred = true
+      self.items[i].locked =  false
+      self.items[i].lockedAmount = ""
+      self.dataChanged(index, index, @[ModelRole.AmountIn.int, ModelRole.ToNetworks.int, ModelRole.HasGas.int,
+        ModelRole.AmountOut.int, ModelRole.IsEnabled.int, ModelRole.IsPreferred.int, ModelRole.Locked.int,
+        ModelRole.LockedAmount.int])
```
Comment (caybro) [inline src/app/modules/main/wallet_section/send/network_model.nim:157]: Ah sorry for not noticing earlier; if you are indeed resetting everything here, you can now even do the simpler:
```qml
self.dataChanged(index, index)
```

**Review by caybro — APPROVED** [review]
LGTM

**Pull request merged**